documentation/src/main/resources/pages/ditto/connectivity-protocol-bindings-hono.md

```diff
@@ -30,20 +33,23 @@ protocol identifier and the host name of `base-uri` to form the connection URI l
 
 Note: If any of these parameters has to be changed, the service must be restarted to apply the new values.
 
-
 ### Source format
 #### Source addresses
 For a Hono connection source "addresses" are specified as aliases, which are resolved at runtime to Kafka topics to subscribe to. 
 Valid source addresses (aliases) are `event`, `telemetry` and `command_response`. 
 Runtime, these are resolved as following:
-* `event` -> `hono.event`
-* `telemetry` -> `hono.telemetry`
-* `command_response` -> `hono.command_response`
+* `event` -> `{%raw%}hono.event.{{connection:id}}{%endraw%}`
+* `telemetry` -> `{%raw%}hono.telemetry.{{connection:id}}{%endraw%}`
+* `command_response` -> `{%raw%}hono.command_response.{{connection:id}}{%endraw%}`
+ 
+Note: The {{connection:id}} will be replaced by the value of connectionId
 
 #### Source reply target
 Similar to source addresses, the reply target `address` is an alias as well. The single valid value for it is `command`. 
 It is resolved to Kafka topic/key like this:
-* `command` -> `hono.command/<thingId>` (&lt;thingId> is substituted by thing ID value).
+* `command` -> `{%raw%}hono.command.{{connection:id}}/<thingId>{%endraw%}` (&lt;thingId> is substituted by thing ID value).
+
+Note: The {{connection:id}} will be replaced by the value of connectionId
```

Comment:
```suggestion
Note: The `{%raw%}{{connection:id}}{%endraw%}` will be replaced by the value of the connection's actual connectionId.
```

This double curly braces also need to be escaped with raw/endraw.
And we should put inline code formatting around them with single back ticks.